src/ui/components/buttons/ButtonPrimaryWhite.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import React from 'react'
+
+import { selectArgTypeFromObject } from 'libs/storybook/selectArgTypeFromObject'
+import { ButtonPrimaryWhite } from 'ui/components/buttons/ButtonPrimaryWhite'
```

Comment:
je recommande d'utiliser les imports comme ce qui suit pour le composant tester dans storybook, comme ça le jour où il faut changer le composant de dossier, le diff sera plus facile à lire 

```suggestion
import { ButtonPrimaryWhite } from './ButtonPrimaryWhite'
```

Reply:
Mmmh d'accord ! Mais pense aussi que peut-être un jour on mettra tout dans un dossier __stories__ 😄

Reply:
j'espère que non